CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ There are breaking changes in this release:
   - Scripts are now located in `/usr/lib/influxdb/scripts` (previously `/opt/influxdb`)
 
 ### Features
+- [#4841](https://github.com/influxdb/influxdb/pull/4841): Improve parsing of measurements and tags
```

Comment:
Can we be a little bit more specific here? The user-visible improvement would be less CPU consumption?, is that right?